advocacy_docs/partner_docs/HashicorpVault/04-ConfiguringHashicorpVault.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
+Implementing Hashicorp Vault with EDB Postgres Advanced Server 15.2 requires the following components installed on the same system:
+
+- EDB Postgres Advanced Server version 15.2 
+- Hashicorp Vault version 1.13.2+ent or 1.12.6+ent
+- [Pykmip](https://pypi.org/project/PyKMIP/#files)
+- Python 
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server version 15.2 instance.
+- Hashicorp Vault installed and deployed per your VM environment
+
+## Configure Hashicorp Vault KMIP Secrets Engine
+
+!!! Note
+    When you are configuring Vault commands and if at any point you receive this error message “Get "https://127.0.0.1:8200/v1/sys/seal-status": http: server gave HTTP response to HTTPS client” you need to issue this in your command line export VAULT_ADDR="http://127.0.0.1:8200".
+
+1. After your Hashicorp Vault configuration is installed and deployed per the guidelines in the [Hashicorp documentation](https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-install), you will then need to enable the kmip capabilities.
+
+2. Assume root user.
+
+3. When you are the root user, type ‘vault secrets enable kmip’ in the command line.
+```bash
+root@ip-172-31-46-134:/home/ubuntu# vault secrets enable kmip
+Success! Enabled the kmip secrets engine at: kmip/
+```
+
+4. You will then need to configure the Hashicorp Vault secrets engine with the desired kmip listener address.
+
+5. Enter `vault write kmip/config listen_addrs=0.0.0.0:5696` in the command line.
+```bash
+root@ip-172-31-46-134:/home/ubuntu# vault write kmip/config listen_addrs=0.0.0.0:5696
+Success! Data written to: kmip/config
+```
+
+6. Define the Scopes and Roles to determine what allowed operations a role can perform.
```

Comment:
steps 6 does not look like a step.  Just a statement.

6. Enter "vault write ....." to create a role that will be used to Define the allowed operations a role can perform.